webui/react/src/components/Grid.module.scss

```diff
@@ -10,3 +10,8 @@
     z-index: 0;
   }
 }
+.row {
+  grid-auto-flow: column;
+  -webkit-overflow-scrolling: touch;
+  overflow-x: auto;
+}
```

Comment:
mobile layout doesnt look right

<img width="467" alt="image" src="https://user-images.githubusercontent.com/109113616/207956951-b4ab4d03-0192-4eb0-a1aa-cd6f1ec39b4c.png">

Reply:
not sure what specific issue you're referring to but I am replacing `<Table>` with `<ResponsiveTable>`

Reply:
text width is too narrow in mobile and table width is wider the screen width

<img width="384" alt="image" src="https://user-images.githubusercontent.com/109113616/207981609-d88dabae-21cb-4ba7-8829-e9b2bdf9a79e.png">

Reply:
how about adding `...`?

like 
<img width="159" alt="image" src="https://user-images.githubusercontent.com/109113616/207985953-f744ef84-9586-4089-8527-8855253de34b.png">
then it would look better in mobile too, but prob @alexjdetermined or @michielpr  have better idea?